### DISCUSSION TITLE
Suggestions about particles and colors!

### DISCUSSION THREAD

**Discussion by ScareLit** [kctekn/obsidian-TagsRoutes#22] [Ideas]
Hello! Wanted to let y'all know how much I love your plugin :D

A few suggestions I have include:
- adding an ability for the graph's nodes to remain stationary in 3D space. It'll make it easier to run on tablets and phones (currently running your plugin with an android tablet). 
- adding a way to toggle the particles running along the node links (also another source of lag on lower end devices)
- a way to change the highlighted text color as opposed to the node color itself for contrast
- a way to turn down the "brightness"? on the nodes? The intensity and bloom messes with more saturated colors

I've been using your plugin to quite literally visualize a story that I'm creating and it's been really helpful in showing how everything connects with each other! Very grateful for this plugin, and thank you for taking the time to read my suggestions :>

**Comment by kctekn** [comment]
Thanks for your feedback and it's great that you love the plugin! The improvements you've suggested can definitely enhance performance, and we'll be testing them out. Implementation might take some time, but these features could be added in a future release. Thanks again for your support, and we look forward to hearing more from you!

**Comment by kctekn** [comment]
Please refer to the new version: https://github.com/kctekn/obsidian-TagsRoutes/releases/tag/1.1.3

**Reply by ScareLit** [reply]
I just took a look, and I am very happy with the results! My tablet doesn't lag anymore lol
Thank you once more for taking the time to read my suggestions and implement them :D
